File: CadastroClienteWebService/Core/CadastroClienteWebService.Domain/Entidades/Cliente.cs
using CadastroClienteWebService.Domain.Enums;
using CadastroClienteWebService.Domain.Validate;
using System;

namespace CadastroClienteWebService.Domain.Entidades
{
    public class Cliente : Pessoa
    {
        public Cliente(int id, string nome, string sobrenome, string cpf, DateTime nascimento, Profissao profissao) 
            : base(id, nome, sobrenome, cpf, nascimento)
        {
            Profissao = profissao;
        }

        public Profissao Profissao { get; private set; }

        public Cliente IsValid()
        {
            new ClienteValidation(this);
            return this;
        }
        public void AdicionarMascaraCpf()
        {
            Cpf = Convert.ToUInt64(Cpf).ToString(@"000\.000\.000\-00");
        }

        public void RemoverMascaraCpf()
        {
            Cpf = Cpf.Replace(".", string.Empty).Replace("-", string.Empty);
        }
    }
}

File: CadastroClienteUI/model/cliente.js
class Cliente{
    constructor(obj){
        this._id = obj.id,
        this._nome = obj.nome,
        this._sobrenome = obj.sobrenome, 
        this._cpf = obj.cpf,
        this._nascimento = obj.nascimento,
        this._profissao = obj.profissao,
        this._idade = obj.idade
    }

    get id(){
        return this._id;
    }

    get nome(){
        return this._nome;
    }

    get sobrenome(){
        return this._sobrenome;
    }

    get cpf(){
        return this._cpf;
    }

    get nascimento(){
        return this._nascimento;
    }

    get profissao(){
        return this._profissao;
    }

    get idade(){
        return this._idade;
    }
}